content/join.hbs

```diff
@@ -85,7 +86,7 @@
 			</div>
 
 			<h1>Join in <span class="semester-text">Spring 2021</span> or later</h1>
-			<p class="has-text-centered">To enroll for a future semester, fill out our interest form and we'll contact you with more details including when and how to apply.</p>
+			<p class="has-text-centered">If you are a college or highschool student, to enroll for a future semester, fill out our interest form and we'll contact you with more details including when and how to apply.</p>

```

Comment:
Same feedback as before. Seems like an unnecessary distinction which makes this sound weird.